### DISCUSSION TITLE
I am not able to upload any image to table

### DISCUSSION THREAD

**Discussion by karanbir-singh09** [AntlerVC/firetable#969] [Support / Q&A] [answered]
![image](https://user-images.githubusercontent.com/114938111/203321266-3c4f3002-a9e5-4795-abed-bb89c9531df0.png)

**Comment by BeeBombshell** [comment, marked as answer]
Hey @karanbir-singh09, it seems like the storage rules didn't get updated with the setup. I have discussed this issue with the team and will hopefully get fixed soon. Meanwhile, you can try to set these rules manually. Navigate to the **Firebase Console > Build > Storage**.

<img width="950" alt="image" src="https://user-images.githubusercontent.com/53828745/203334963-b3584faa-5de9-4103-8db4-6ba4350503cf.png">

From there, you'll be able to set the rules for Firestore Storage. Paste the following rules and click **Publish:**
```
rules_version = '2';
service firebase.storage {
  match /b/{bucket}/o {
    match /{allPaths=**} {
      allow read, write: if request.auth.token.roles.size() > 0;
    }
  }
}
```

<img width="580" alt="image" src="https://user-images.githubusercontent.com/53828745/203335515-89cf77ea-99fe-46e4-ab07-7b76152e8502.png">
